app/components/Avatar/AvatarWithPresence.js

```diff
@@ -37,20 +39,28 @@ class AvatarWithPresence extends React.Component<Props> {
       isPresent,
       isEditing,
       isCurrentUser,
+      t,
     } = this.props;
 
     return (
       <>
         <Tooltip
           tooltip={
             <Centered>
-              <strong>{user.name}</strong> {isCurrentUser && "(You)"}
-              <br />
-              {isPresent
-                ? isEditing
-                  ? "currently editing"
-                  : "currently viewing"
-                : `viewed ${distanceInWordsToNow(new Date(lastViewedAt))} ago`}
+              {t(
+                "<strong>{{ userName }} {{ you }} <br /> {{ action }}</strong>",
```

Comment:
We don't want to include JSX in this strings, right?

Reply:
This double curly brackets are not JSX, those are only placeholders for the t function to render the variables dynamically.

![grafik](https://user-images.githubusercontent.com/17197309/98943095-45194600-24ef-11eb-9650-cea89b3a3159.png)

Reply:
The HTML code is ok to be included

Reply:
gonna fix it